docs/contributing/contributing-releases/README.md

```diff
@@ -1,116 +1,158 @@
 
-In the project-radius/samples repo, run the [Test Samples](https://github.com/project-radius/samples/actions/workflows/test.yaml) workflow. Run the workflow from the edge branch and using the Radius release or pre-release version number being released.
+Conceptually we scope channels to a major+minor pair because this allows us to freely patch assets as needed without needing to change the intermediate pieces. For example pushing a `v0.1.1` tag will update the assets in the `v0.1` channel. This works as long as it is a *true* patch release and maintains compatibility.
 
-> For now, this is a manual task. Soon, this workflow will be triggered automatically.
-> There is a possibility that the workflow run failed because of flaky tests. Try re-running, and if the failure is persistent, then there should be further investigation.
+## Cadence
 
-If this workflow run fails, or if we encounter an issue with an RC release, please refer to "Patching" below.
+We follow a monthly release cadence. Any contributions that have been merged through the pull-request process will be present in the next scheduled release.
 
-### Creating the final release
 
-If sample validation passes, we can start the process of creating the final release.
+## Release Process
 
-1. In the `radius-project/bicep` repo, in the release branch, change the `version.json` version to the new release number. Create a pull request and merge this change into the bicep-extensibility and release/\<release-version\> branch.
+### Creating an RC release
 
-1. Go through steps 1-4 of "Creating an RC release" above, substituting the final release version instead of the RC version. For example, if the RC version number is `0.1.0-rc1`, the final release version would be `0.1.0`.
+When starting the release process, we first kick it off by creating an RC release. If we find issues in validation, we can create additional RC releases until we feel confident in the release.
 
-1. In this PR, create a new release note document in the [release-notes](../../release-notes/) directory using the automatically generated release notes comment. Follow the directory's README.md for instructions on how to create a new release note document. Include this file in the release version pull request. [Example](https://github.com/project-radius/radius/pull/6092/files)
+Follow the steps below to create an RC release.
 
-1. Create a PR to merge into the release branch (format: release/x.y) in the radius repo. Cherry-pick the `version.yaml` changes and the release notes from the previous step in this PR. This will ensure that the version changes and release notes are included in the release branch. [Example](https://github.com/radius-project/radius/pull/6114/files)
+1. Clone the [radius-project/radius](https://github.com/radius-project/radius) repo locally, or use your existing local copy.
+   ```
+   git clone git@github.com:radius-project/radius.git
```

Comment:
Should this be a fork rather than the main repo? Are we counting on the release captain to have push permissions?

Reply:
Yes, for now we are counting on the release captain to have push permissions at the moment. This is because we require the GH token to create a comment on the PR. We could definitely change this but for now we should fork directly from the repo.

ref: https://github.com/radius-project/radius/issues/6927#issuecomment-1944388891